docs/interactions/Receiving_and_Responding.md

```diff
@@ -78,7 +99,6 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
 \* Partial `Member` objects are missing `user`, `deaf` and `mute` fields
 
 \*\* Partial `Channel` objects only have `id`, `name`, `type` and `permissions` fields. Threads will also have `thread_metadata` and `parent_id` fields.
-
```

Comment:
Very small thing, but typically a blank line exists before each header so I think this should stay